site/content/en/blog/releases/1.23.0.md

```diff
@@ -0,0 +1,64 @@
+
+* Fix for health controller race condition marking healthy pods as unhealthy.
+* Check for DeletionTimestamp of fleet and gameserverset before scaling.
+* GameServer stucks in Shutdown state preventing a rolling update to complete.
+* Fix for GameServer back to Ready always.
+* Queue updated GameServers with DeletionTimestamp.
+
+Check the <a href="https://github.com/googleforgames/agones/tree/release-1.23.0" data-proofer-ignore>README</a> for details on features, installation and usage.
+
+**Breaking changes:**
+
+- Upgrade terraform to Kubernetes 1.22 [\#2551](https://github.com/googleforgames/agones/pull/2551) ([roberthbailey](https://github.com/roberthbailey))
+
+**Implemented enhancements:**
+
+- Upgrade to Kubernetes 1.22 [\#2494](https://github.com/googleforgames/agones/issues/2494)
+- Update golang version used in App Engine  [\#2382](https://github.com/googleforgames/agones/issues/2382)
+- Allocator controller arm64 img [\#2565](https://github.com/googleforgames/agones/pull/2565) ([Ludea](https://github.com/Ludea))
+- sdk arm64 images [\#2521](https://github.com/googleforgames/agones/pull/2521) ([Ludea](https://github.com/Ludea))
+
+Documentation: https://development.agones.dev/site/
+
+See <a href="https://github.com/googleforgames/agones/blob/release-1.23.0/CHANGELOG.md" data-proofer-ignore>CHANGELOG</a> for more details on changes.
+
+Images available with this release:
+
+- [gcr.io/agones-images/agones-controller:1.23.0](https://gcr.io/agones-images/agones-controller:1.23.0)
+- [gcr.io/agones-images/agones-sdk:1.23.0](https://gcr.io/agones-images/agones-sdk:1.23.0)
+- [gcr.io/agones-images/agones-ping:1.23.0](https://gcr.io/agones-images/agones-ping:1.23.0)
+- [gcr.io/agones-images/agones-allocator:1.23.0](https://gcr.io/agones-images/agones-allocator:1.23.0)
+- [gcr.io/agones-images/allocation-endpoint-proxy:0.1](https://gcr.io/agones-images/allocation-endpoint-proxy:0.1)
+- [gcr.io/agones-images/autoscaler-webhook:0.4](https://gcr.io/agones-images/autoscaler-webhook:0.4)
+- [gcr.io/agones-images/cpp-simple-server:0.14](https://gcr.io/agones-images/cpp-simple-server:0.14)
+- [gcr.io/agones-images/crd-client:0.6](https://gcr.io/agones-images/crd-client:0.6)
+- [gcr.io/agones-images/nodejs-simple-server:0.6](https://gcr.io/agones-images/nodejs-simple-server:0.6)
+- [gcr.io/agones-images/rust-simple-server:0.10](https://gcr.io/agones-images/rust-simple-server:0.10)
+- [gcr.io/agones-images/simple-game-server:0.12](https://gcr.io/agones-images/simple-game-server:0.12)
+- [gcr.io/agones-images/supertuxkart-example:0.4](https://gcr.io/agones-images/supertuxkart-example:0.4)
+- [gcr.io/agones-images/unity-simple-server:0.3](https://gcr.io/agones-images/unity-simple-server:0.3)
+- [gcr.io/agones-images/xonotic-example:0.8](https://gcr.io/agones-images/xonotic-example:0.8) 
+- [gcr.io/agones-images/cpp-simple-server:{example-version}](https://gcr.io/agones-images/cpp-simple-server:{example-version})
```

Comment:
Lines 54-57 should be removed (they are redundant with the above lines).

Reply:
These lines cause failures in the CI tests:

```
  Non-OK status: 404 --- site/blog/2022/05/10/1.23.0-kubernetes-1.22-sdk-arm64-images-and-important-bug-fixes/index.html --> https://gcr.io/agones-images/cpp-simple-server:%7Bexample-version%7D
  Non-OK status: 404 --- site/blog/2022/05/10/1.23.0-kubernetes-1.22-sdk-arm64-images-and-important-bug-fixes/index.html --> https://gcr.io/agones-images/crd-client:%7Bexample-version%7D
  Non-OK status: 404 --- site/blog/2022/05/10/1.23.0-kubernetes-1.22-sdk-arm64-images-and-important-bug-fixes/index.html --> https://gcr.io/agones-images/nodejs-simple-server:%7Bexample-version%7D
  Non-OK status: 404 --- site/blog/2022/05/10/1.23.0-kubernetes-1.22-sdk-arm64-images-and-important-bug-fixes/index.html --> https://gcr.io/agones-images/rust-simple-server:%7Bexample-version%7D
```

which probably would have been more obvious if the other link checking failures hadn't recently cropped up.